org.gnome.Keysign.yml

```diff
@@ -0,0 +1,306 @@
+id: org.gnome.Keysign
+branch: master
+runtime: org.gnome.Platform
+runtime-version: '3.28'
+sdk: org.gnome.Sdk
+command: gnome-keysign
+copy-icon: true
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --share=network
+  - --socket=system-bus
```

Comment:
Why does it need access to entire system bus?

Reply:
Now I listed only the single bus we need (Avahi).